system_tests/test_cli_package.py

```diff
@@ -241,54 +241,41 @@ def bootstrap_manager(self):
         if self.is_install_plugins():
             install_plugins = '--install-plugins'
         out = self._execute_command(
-            'bootstrap -p {0} -i {1} {2}'.format(
-                self.test_manager_blueprint_path,
-                self.remote_bootstrap_inputs_path,
+            """bootstrap -p {0} -i "{1}" {2}""".format(
```

Comment:
also for future reference, triple quotes is for multiline strings. you should probably just alternate between regular quotes and single quotes (" and ') or use escaping here.

also note that you're passing shell arguments here, so it'd make more sense for {1} to be surrounded with single quotes ('), since that tells the shell to take the string as-is, as opposed to quotes (") which the shell might interpret differently.